src/paperless/version.py

```diff
@@ -1 +1 @@
-__version__ = (1, 5, 0)
+__version__ = (1, 6, 0)
```

Comment:
Should this change be noted in the [org docs](https://github.com/paperless-ngx/org-docs/blob/main/dev-release-cycle.md)?

Reply:
Oh yea I forgot to mention this, my bad. I noticed it a little while ago, and yes I added to docs!

Reply:
Awest